cumulusci/cumulusci.yml

```diff
@@ -781,6 +798,7 @@ flows:
                 task: uninstall_packaged_incremental
 
     dev_org:
+        group: Org Creation
```

Comment:
This is nitpicking but how about Org Setup. Technically the org is created before we run any flows on it.